pulp_python/tests/functional/api/test_crd_publications.py

```diff
@@ -6,8 +6,8 @@
 from pulp_smash.tests.pulp3.constants import REPO_PATH, DISTRIBUTION_PATH, PUBLICATIONS_PATH
 from pulp_smash.tests.pulp3.utils import gen_distribution, gen_repo, get_auth, publish, sync
 
-from pulp_python.tests.functional.constants import (PYTHON_PUBLISHER_PATH, PYTHON_PYPI_URL,
-                                                    PYTHON_REMOTE_PATH)
+from pulp_python.tests.functional.constants import (
+    PYTHON_PUBLISHER_PATH, PYTHON_PYPI_URL, PYTHON_REMOTE_PATH)
```

Comment:
Thoughts on this way vs. vertical listing?

Reply:
I tend to prefer vertical listing since it's a lot easier for me to scan for whatever it is that I'm looking for

Reply:
vertical is also easier to keep it in alphabetical order, and for changing only 1 line in a commit. I don't feel strongly though.

Reply:
Changing in next push